lib/python/qmk/commands.py

```diff
@@ -61,3 +64,22 @@ def parse_configurator_json(configurator_file):
     user_keymap = json.load(configurator_file)
 
     return user_keymap
+
+
+def run(command, **kwargs):
+    """Run a command with subprocess.run
+    """
+    OS = platform.platform().lower()  # noqa (N806), uppercase name is ok in this instance
+
+    if 'windows' in OS:
+        # Make sure the command is a string
+        if isinstance(command, list):
+            command = ' '.join(command)
+        # On Windows, we want to run stuff (especially scripts)
+        # in a Posix shell
+        command = [os.environ['SHELL'], '-c', command]
```

Comment:
I think we should explicitly use /bin/sh here because users might be using tcsh or another shell that isn't compatible.

```suggestion
        command = ['/bin/sh', '-c', command]
```